### PR TITLE
Translate "Your GDPR Rights" page into French

### PR DESCRIPTION
**Pull request by NathanBnm**
I translated this page into French but as the GDPR has been applied I think this page should be updated no?

### CODE REVIEW

File: content/blog/your-gdpr-rights.fr.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+> L'évolution rapide des technologies et la mondialisation ont créé de nouveaux enjeux pour la protection des données à caractère personnel. L'ampleur de la collecte et du partage de données à caractère personnel a augmenté de manière importante. Les technologies permettent tant aux entreprises privées qu'aux autorités publiques d'utiliser les données à caractère personnel comme jamais auparavant dans le cadre de leurs activités. De plus en plus, les personnes physiques rendent des informations les concernant accessibles publiquement et à un niveau mondial. Les technologies ont transformé à la fois l'économie et les rapports sociaux, et elles devraient encore faciliter le libre flux des données à caractère personnel au sein de l'Union et leur transfert vers des pays tiers et à des organisations internationales, tout en assurant un niveau élevé de protection des données à caractère personnel.
```
Comment (baltpeter) [inline content/blog/your-gdpr-rights.fr.md]: In my version, I have replaced the sentence "Les technologies permettent tant aux entreprises privées qu'aux autorités publiques d'utiliser les données à caractère personnel comme jamais auparavant dans le cadre de leurs activités." with `[…]` as it wasn't important to the point I am trying to make.

I'll leave it up to you whether you also want to do that.

**Comment by baltpeter** [comment]
> I translated this page into French but as the GDPR has been applied I think this page should be updated no?

Good point, I'll update the German and English posts.

File: content/blog/your-gdpr-rights.fr.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+Dans cet article, nous voulons te donner un aperçu détaillé des droits dont tu disposes grâce au RGPD. Si tu recherches plutôt un bref résumé, consulte [cet article]({{< ref "/blog/a-quick-overview" >}}).
+
+## Qu'est-ce que le RGPD ?
+
+Le RGPD est un *règlement de l'UE* qui entrera en vigueur le 25 mai 2018. L'objectif du RGPD est de donner aux citoyens de l'UE un meilleur contrôle sur leurs données personnelles. En uniformisant la réglementation sur la protection des données dans toute l'UE, la situation juridique des citoyens et des entreprises sera plus claire et la bureaucratie réduite.
+
+Le RGPD remplace principalement la législation nationale précédente sur la protection des données, comme la loi britannique *Data Protection Act 1998* (*DPA*) (dans certains aspects, il permet toutefois aux États membres une interprétation individuelle dans la législation nationale. En tant que règlement de l'UE, il est directement applicable dans tous les États membres de l'UE, sans que ceux-ci aient à le transposer dans leur droit national.
```
[the remark this answers is not in the document]
Comment (NathanBnm) [reply content/blog/your-gdpr-rights.fr.md]: Sure it's the "Informatique et Libertés" law. The RGPD surpasses this law, but according to the CNIL (the French supervisory authority) it has been reworked and is still applicable. But I think that's what you were implying with the parenthesis (one is missing btw).

[the remark this answers is not in the document]
Comment (NathanBnm) [reply content/blog/your-gdpr-rights.fr.md]: Source is here: https://www.cnil.fr/fr/la-loi-informatique-et-libertes

File: content/blog/your-gdpr-rights.fr.md (hunk 1 above)
Comment (NathanBnm) [reply content/blog/your-gdpr-rights.fr.md]: I'll do that

**Comment by NathanBnm** [comment]
> Good point, I'll update the German and English posts.

Notify me if the French translation has to be updated afterwards.

**Comment by baltpeter** [comment]
Combined into #237.

**Pull request closed without merging**